### PR TITLE
Switch codesigning to use Eclipse Foundation certificate

### PR DESCRIPTION
**Pull request by gdams**
Currently this only works for Windows and macOS (jdk8u). I will need to fix up the remaining macOS versions.

### CODE REVIEW

File: sign.sh [hunk 1]
```diff
@@ -104,16 +113,30 @@ signRelease()
...
+          chmod "$permissions" "$f"
+          rm -rf "${dir}/unsigned_${file}"
+        done
+      else
+        # Login to KeyChain
+        # shellcheck disable=SC2046
+        # shellcheck disable=SC2006
+        security unlock-keychain -p `cat ~/.password` login.keychain-db
+        xattr -cr .
+        echo "$FILES" | while read -r f; do codesign --entitlements "$ENTITLEMENTS" --options runtime --timestamp --sign "Developer ID Application: London Jamocha Community CIC" "$f"; done
```
Comment (karianna) [inline sign.sh:138]: Swap LJC to Eclipse here?

File: sign.sh [hunk 2]
```diff
@@ -154,6 +177,10 @@ configDefaults
 parseArguments "$@"
 extractArchive
 
+if [ "${OPERATING_SYSTEM}" == "windows" ]; then
+  BUILD_CONFIG[OS_KERNEL_NAME]="cygwin"
```
Comment (karianna) [inline sign.sh:181]: Is it always cygwin?  Why are we doing this?

File: build-farm/platform-specific-configurations/mac.sh [hunk 3]
```diff
@@ -65,17 +65,17 @@ else
...
-  # shellcheck disable=SC2046
-  # shellcheck disable=SC2006
-  security unlock-keychain -p `cat ~/.password` login.keychain-db
-  rm -rf codesign-test && touch codesign-test
-  codesign --sign "Developer ID Application: London Jamocha Community CIC" codesign-test
-  codesign -dvvv codesign-test
-  export BUILD_ARGS="${BUILD_ARGS} --codesign-identity 'Developer ID Application: London Jamocha Community CIC'"
+  ## Login to KeyChain
+  ## shellcheck disable=SC2046
+  ## shellcheck disable=SC2006
```
Comment (karianna) [inline build-farm/platform-specific-configurations/mac.sh:73]: Add a TODO here

**Review by karianna — CHANGES_REQUESTED** [review]

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**